client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -223,7 +223,7 @@ object FlightTableRow {
         hasChangedStyle,
         props.timelineComponent.map(timeline => <.td(timeline(flight))).toList.toTagMod,
         flightFields.toTagMod,
-        props.splitsQueueOrder.map(q => <.td(<.span(s"${queuePax.getOrElse(q, 0)}"), ^.className := s"queue-split $paxClass right")).toTagMod
+        props.splitsQueueOrder.map(q => <.td(<.span(s"${queuePax.getOrElse(q, 0)}"), ^.className := s"queue-split $paxClass ${q.toString.toLowerCase()}-queue-pax right")).toTagMod
```

Comment:
what's the deal with these new css classes?

Reply:
they are just to make the e2e tests more semantic.